draft-sharabayko-mops-srt.md

```diff
@@ -1249,6 +1239,25 @@ of the handshake Extension Message Flags to 1.
 In this mode consecutive packets form one continuous stream that can be read, with 
 portions of any size.
 
+### Use Case Live Streaming {#use-case-live-streaming}
+
+This section describes real world examples of live audio/video streaming and the current consensus, which is maintaining compatibility between SRT implementations by different vendors. It's meant as guidance for developers to be compatible with existing SRT implementations.
+
+The term "live streaming" refers to MPEG-TS style continuous data transmission with latency management. Live streaming based on segmentation and transmission of files like in HLS protocol is not part of this use case.
+
+The default mode for continuous live streaming is Message Mode. 
```

Comment:
> The default mode for continuous live streaming is Message Mode. 

The default mode for continuous live streaming is Message Mode with certain settings described below.

or something like this